OCR/ocr/api.py

```diff
@@ -22,10 +22,10 @@ async def health_check():
 
 @app.post("/image_to_text/")
 async def image_to_text(source_image: UploadFile, segmentation_template: UploadFile, labels: str = Form()):
-    source_image_np = np.fromstring(await source_image.read(), np.uint8)
+    source_image_np = np.frombuffer(await source_image.read(), np.uint8)
```

Comment:
Is this relevant to the labels.json or is it just more accurate to the type of data being sent?

Reply:
It's not relevant, but I noticed that `fromstring` was deprecated and they recommend switching to `frombuffer`